### PR TITLE
add groups_of_similar_descriptive_words.pdf to main page

### PR DESCRIPTION
**Pull request by eve-git**
*Issue #:* /bcgov/entity#12017

*Description of changes:*

1. add the pdf file to public folder
2. add an url point to the pdf file to the "Check for similar names in the same sector"

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Please:
1. rebase to resolve the conflicts
2. add yourself as "assignee"
3. add another reviewer

**Comment by severinbeauvais** [comment]
Please:
- move the ticket to Review pipeline
- update the checklist in the ticket to reflect the latest requirements (and check off what you've done)
- add any helpful test notes in the ticket
- attach a screenshot for verification?

File: src/components/lower-info-area/name-build-sub-components/unique-names.vue [hunk 1]
```diff
@@ -31,6 +31,9 @@
         <strong>Westcoast Barbers Inc.</strong>
       </p>
     </v-row>
+    <v-row class="ml-1 mb-4">
```
Comment (severinbeauvais) [inline src/components/lower-info-area/name-build-sub-components/unique-names.vue:34]: As this is a new row, I recommend whitespace (a blank line) to make it separate from the other rows.

File: src/components/lower-info-area/name-build-sub-components/unique-names.vue [hunk 2]
```diff
@@ -31,6 +31,9 @@
...
+      <a href="./pdfs/groupings_of_similar_descriptive_words.pdf"> Groupings of similar descriptive words</a>
```
Comment (severinbeauvais) [inline src/components/lower-info-area/name-build-sub-components/unique-names.vue:35]: Why do you have a space in front of the text?

Comment (eve-git) [reply src/components/lower-info-area/name-build-sub-components/unique-names.vue]: It was a mistake. corrected.

Comment (severinbeauvais) [reply src/components/lower-info-area/name-build-sub-components/unique-names.vue]: OK. If it was intentional then I had a comment about it.

Resolved.

**Review by severinbeauvais — APPROVED** [review]
Thanks for the updates. Looks good!

Thanks especially for the screenshot in the ticket -- it will help with QA.

**Comment by eve-git** [comment]
@severinbeauvais I updated the code upon Yui's comments. I supposed to contact Yui before code change. Sorry about it. Can you check for my latest code change? Thanks!

**Pull request merged**